USER_MANUAL.MD

```diff
@@ -0,0 +1,5 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### How to add a gym 
```

Comment:
What do we define as a gym? What are the directories and files which make a gym?